sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="Electron" modified="2021-07-21T08:03:58.207Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/14.6.13 Chrome/89.0.4389.128 Electron/12.0.7 Safari/537.36" etag="6XXRi8pDeGpdp_IAuZz0" version="14.6.13" type="device"><diagram id="kgpKYQtTHZ0yAKxKKP6v" name="Page-1">7Vlbc6M2FP41zLQP9ggEGB59TWc2m80k7SbZNxlkzAYjKuTb/vpKIK7Csd046aatXywdiSNxvnP5JDQ4Xu2uKEqWn4mPI80A/k6DE80wdOA4/E9I9rnEcUAuCGjoy0mV4D78gYsnpXQd+jhtTGSERCxMmkKPxDH2WEOGKCXb5rQFiZqrJijAiuDeQ5EqfQh9tpRvYYFK/hsOg2Wxsg7kyAoVk6UgXSKfbGsiONXgmBLC8tZqN8aRMF5hl/y52YHRcmMUx+yUB5wvCH2iN8AlT18njz9csnke9sxcywZFa/nCcrNsX1iAknXsY6EEaHC0XYYM3yfIE6NbjjmXLdkq4j2dN32ULrO5RecWMYZpnEkMIKTqvuWrbDBleFcTyfe4wmSFGd3zKXLUcAb5I9KpTLnrbYWQbsopyxo6piG9EUmvCErVleF4Q9qu244wXo9mO+p6+2uQxF8efjfQVU9X7PiA51wwTJIo9BALScx7hGoCJFszoC6MeYdTsuaWNMA9pvz1Fdtzn0lEc72KrsMFjsKY90YJpiHfNp8PJ5EU31ayEY8FhriMShw8EkUoScN5plYsTLG3pmm4wXwHechl0hJp8VTpr2IoZZQ8lxEghhdhFI1JRGi2UzibjUajC8GrFyFT5AygwqubKrywmHhxeI3jYZKQMGbZutZIsyYtmAhlSxKQGEV1oJqhdcTgqmFf9MSTrW0ZDWPbHcYGHbYGb2Vr60AooXoofdg4cfTxcDq7TJxAXT8aJ6ALuzeLE/vjxIl1rrVN6x+LE2KxPzY3n3ZPTry04Of95OYe94oMWLMt9jl3kd3KktNK2rJkNeeakETa7ztmbC+JGFoz0izw3Kh0/1jvPAllfXPgFoLJTqrPe/t6r4ZqJlzwEJRr6eeDJ973RegojnjC2DS5XBcS8tFb4Zy11NiEXB+APq9N1c9uKmSIBphJHXUmdkSt2dLDSQH1sKJnSCna16bJSDq4+zL2W+sc2pYyf9AglLyR76By2dLSr/DigZIyvqIo9BETtAh5Hk7TjPM/YzXtC/yv0ZwfOeCIPxQImulxz8mShohsXi+ioRxYhb6fx4DI5ahK8mpGOu6VL0akklPK44lcVaufALpyTQ/0bRdKt3ilEztmf2A3gW3pIItFipnWTkjnodsZpyq4d5itqeDCqSitPBPxoyFiSIG2jd6cMEZWWRrxh+JcJ1Jcwn2idebgwxI5R/R2IXss8g9v57nKtU4o20ddQH+fHKQ7sAEedK2+W//9vRwELfsctQdS0qX8xFH8ZFhEPo9WRkOPZa5Ccb6RVBsb2rB+lCKL2nA+mnafqU7xrHlEvOcP5SRNEq8PBifhd9QtFEUH/OtS5UBNGBzjiAnrJyhuAGn/uRZXJ6M58p6DjNf0vJxnD4VBg/kvWaHl+wCGSKWiodvurxlOQEDXyzm9mK7ryS4bKLTyViD+9WJ9/jr5FnK54lZnXZNEomiNyo3Xzgfm0LFns67z9XQ8dEsvLEZikp10TnbEMw7hlt7yBJmH6oRXxm2d8OoXuGLpdI3yHu3fzXffKWNAu5X/od3gti2FJ5eVtlrd6ANwmDJfiOqarZMwX/bFbSrzYYMavw3VNdRLwqlo51SXn4JXYZrmF4VzlGb1rl7gYlnVPE570n5nLLwHF+ZJ0Avj4E5GPKhE13ghJLp9OmMuYvrnYswuaDNmt98mRBfhzN026bqTf10NBLUyCGqV0ACHi6FSCeH/lVAmC8d+t0rY+dFGvSG9K4jvwS8J/F1Z0/Ld5qvZWopOTyZdUDed4Z0+D7lmv3nFYnYgBo1iUh004wLXddSm5rfv8TcXBDMY956MrYs7vhD9FFFt/FejurzAlS5iFN8GLx/VvFt95s3LRPWxHE7/Ag==</diagram></mxfile>
```

Comment:
![image](https://user-images.githubusercontent.com/13167207/126455428-2c065c94-e0f3-41bb-8190-460f607b9f3c.png)

Reply:
why do I feel that the image is not ver clear @chenrujun